Shared/Samples/Navigate route with rerouting/README.md

```diff
@@ -0,0 +1,57 @@
+# Navigate route with rerouting
+
+Navigate between two points and dynamically recalculate an alternate route when the original route is unavailable.
```

Comment:
```suggestion
Navigate between two points and dynamically recalculate an alternate route when the original route is not followed.
```
A little clearer to me, but just a suggestion.

Reply:
I think we generally try to leave the sample's description the same, since it is shared between the different platforms.

Reply:
That's a good reason to leave it as-is.